hubmap/frontend/src/components/Experiments.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import grey from '@material-ui/core/colors/grey';
+import PropTypes from 'prop-types';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+
+const StyledTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: grey[800],
+    color: theme.palette.common.white,
+  },
+  body: {
+    fontSize: 14,
+  },
+}))(TableCell);
+
+const StyledTableRow = withStyles(theme => ({
+  root: {
+    '&:nth-of-type(odd)': {
+      backgroundColor: grey[300],
+    },
+  },
+}))(TableRow);
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    overflowX: 'auto',
+  },
+  table: {
+    minWidth: 700,
+  },
+}));
+
+class Experiments extends React.Component {
```

Comment:
I think it's preferred to just use functions, unless you need the class-based style for some reason?

Reply:
> I've just looked at the top of this, and I think there are lots of little things which could be done differently... None of these should block a merge, but they should be addressed at some point, both for their own sake, and to get into good habits going forward.
> 
> I would suggest that you (@sushma-ananda and @mruffalo) get together and decide how to merge this... but if you could identify some files that could use a fine-grained post-hoc review, I would appreciate that.

@mccalluc but you already gave some comments on this pull request . I appreciate your critical review and sharing your thoughts, but I have replied to the comments.